spinnaker-monitoring-daemon/spinnaker-monitoring/__main__.py

```diff
@@ -157,7 +160,7 @@ def main(config_search_path):
   stores = options['monitor'].get('metric_store', [])
   if not isinstance(stores, list):
     stores = [stores]
-  stores.extend([store for store in ['datadog', 'prometheus', 'stackdriver']
+  stores.extend([store for store in ['datadog', 'prometheus', 'stackdriver', 'newrelic']
```

Comment:
🤔 Can you pull this out into a constant or something? I don't like that there's a list just floating out there in some method that we have to change.